### PR TITLE
20801:  Makes API changes in preparation for apiv3, MAJOR

### PR DESCRIPTION
**Pull request by jackx111**

### CODE REVIEW

File: howso/attributes.amlg [hunk 1]
```diff
@@ -949,16 +949,16 @@
 	;example: (assoc "day" 7 "degrees" 360)
 	#!SetCyclicFeatures
 	(declare
-		(assoc features (assoc))
-		(if (= 0 (size features))
+		(assoc feature_attributes (assoc))
```
Comment (cademack) [inline howso/attributes.amlg:952]: update the "docstring" above the method with the new parameter name

File: howso/details.amlg [hunk 2]
```diff
@@ -71,7 +71,7 @@
 	;						model area. Uses only the context features of the reacted case to determine that area.
 	;
 	;		 "categorical_action_probabilities" true or false. If true outputs probabilities for each class for the action
-	;						features. Applicable only to categorical action features. react_series() calls additionally
+	;						features. Applicable only to categorical action features. single_react_series() calls additionally
```
Comment (cademack) [inline howso/details.amlg]: can we drop the parenthesis here? these definitely don't make sense here in Amalgam land

File: howso/react_series_utilities.amlg [hunk 3]
```diff
@@ -197,12 +197,12 @@
...
 	;
 	;parameters:
-	;	see comment block for #react_series in howso.amlg for details
+	;	see comment block for #single_react_series in howso.amlg for details
 	#!ReactSeries
 	(seq
 
 		(if (= 0 (size output_features))
-			;TODO: 15300 feed into react_group() and output results if details are provided
+			;TODO: 15300 feed into single_react_group() and output results if details are provided
```
Comment (cademack) [inline howso/react_series_utilities.amlg]: can we remove the parenthesis here too

File: howso/react_utilities.amlg [hunk 4]
```diff
@@ -42,7 +42,7 @@
 		)
 	)
 
-	;Generate lists of unique values for each 'unique' feature that isn't being conditioned on for use in #React
+	;Generate lists of unique values for each 'unique' feature that isn't being conditioned on for use in #single_react
```
Comment (cademack) [inline howso/react_utilities.amlg]: should this be #SingleReact? seems to have changed from referencing the private label to the public label

File: migrations/migrations.amlg [hunk 5]
```diff
@@ -271,5 +271,5 @@
 
 		(assign_to_entities updated_model_atributes_map)
 	)
-
-))
\ No newline at end of file
+"80.0.0"
```
Comment (cademack) [inline migrations/migrations.amlg]: what's this for? this makes it an invalid assoc

Comment (cademack) [reply migrations/migrations.amlg]: well invalid might be a strong word, it'll just have this key with a (null) value

Comment (jackx111) [reply migrations/migrations.amlg]: whoops, at a certain point i thought i needed this, was playing around

**Review by cademack — DISMISSED** [review]

**Review by fulpm — DISMISSED** [review]

**Review by cademack — APPROVED** [review]

**Pull request merged**